api/package.json

```diff
@@ -32,10 +33,10 @@
     "name": "A Strapi developer"
   },
   "strapi": {
-    "uuid": "0ce2c844-73f4-41fa-a986-9f50226d1ce9"
+    "uuid": "FOODADVISOR-LOCAL-91bcb9c3-39e5-43e4-ac15-05df0813913f"
   },
   "engines": {
-    "node": ">=12.x.x <=16.x.x",
+    "node": ">=12.x.x <=18.x.x",
```

Comment:
Do we still need node 12 here then?

Reply:
Should we set the min to 14?

Reply:
Well, Strapi is now officially 14 to 18, and since you've bumped up FoodAdvisor to Strapi 4.5, I think it makes sense to update the node version accordingly. Not that it's gonna make much of a difference or break anything, just following the main product requirements

Reply:
Sure! Will do that!